migcontrol/templates/base.html

```diff
@@ -5,17 +5,23 @@
 
 <!DOCTYPE html>
 <html class="no-js" lang="en">
+
 <head>
-  <meta charset="utf-8" />
+<style>
+<style>
+a.hover {
+  background-color: yellow;
+}
+</style>
+</style>
+
```

Comment:
It's unclear why all this code is here and why you have removed `<meta charset="utf-8" />`